routes/articles.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express")
+const router = express.Router()
+const Article = require('../models/article')
+
+//put 1 article
+router.post('/', async (req,res) => {
+
+    const article = new Article({
+        title : req.body.title,
+        description :  req.body.description,
+        author : req.body.author,
+        tags :  req.body.tags
+    })
+    try {
+        const newArticle = await article.save()
+        //201 cause its sucess
+        res.status(201).json(newArticle)
+    } catch(arr) {
+        res.status(400).json( { message : err.message })
+    }
+})
+
+
+//get all articles
+router.get('/' , async (req,res) => {
+    try {
+        const articles = await Article.find()
+        res.json(articles)
```

Comment:
why no `status` here ,

If we are doing a check in frontend `if (response.status)`
and this would not have response 

Although I am assuming it would have some default value I

Reply:
Ithink that should be added after more search options are available, we need to check if the error was caused by server or the user's search option.

Reply:
This pr was made so that the initial files are available for others to add upon, since for now the repo is empty